scylla/src/transport/partitioner.rs

```diff
@@ -12,7 +12,9 @@ pub(crate) enum PartitionerName {
 
 impl PartitionerName {
     pub(crate) fn from_str(name: &str) -> Option<Self> {
-        if name.ends_with("CDCPartitioner") {
```

Comment:
Commit 441ac1dfbe9f85bc0524c6fe310ac2589f27eb2a doesn't compile.
Please make sure that 
>  All commits compile, pass static checks and pass test.

We want to be able to use `git bisect` to test various versions and quickly find regressions.

It would also be nice to squash fixup comments into their original versions, it would give us a clean change history that's easier to review.

Reply:
I've rebased to make all commits pass the tests.

However, I'm not sure to understand
> It would also be nice to squash fixup comments into their original versions, it would give us a clean change history that's easier to review.

Should I squash 546a890, d93bba4 and b3b4ca2 into fe8bd3c ?

Reply:
In this project we usually edit commit history to address review comments instead of pushing new changes.
This allows us to have a nice clean readable commit history.
Reviewing rebased branch isn't as bad as it might seem, github has utilities to show the diff.
It's a preference thing, I know that some people dislike rebasing public branches.

Squashing the commits that edit previous commits would make the history nicer, but we can live without it.

Reply:
I'm more used to squash&merge, but I gladly adapt.